_projects/2024/100453237/100453237.Rmd

```diff
@@ -0,0 +1,589 @@
+---
+title: "Ternary graph by ‘The Economist’: The world’s most, and least, walkable cities"
+description: |
+  Visual representation of commuting patterns in cities worldwide, highlighting the proportions of journeys made by car, active mobility (such as walking and cycling), and public transport. The data is based on the study 'The ABC of Mobility' by Rafael Prieto-Curiel and Juan P. Ospina.
+categories: "2024"
+author: Pablo Aísa
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true 
+---
+
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(echo = TRUE, fig.showtext = TRUE, 
+                      fig.dpi = 150, warning = FALSE, 
+                      out.width="100%", fig.align="center")
+```
+
+## Introduction
+
+The Economist article reinterprets data from Rafael Prieto-Curiel and Juan P. Ospina's study on mobility worldwide, [*The ABC of mobility*](https://www.sciencedirect.com/science/article/pii/S0160412024001272). The study attempts to study the mobility of the inhabitants of 794 cities by grouping the types of transport into a triplet: A (active mobility), B (bus, public transport), and C (private car).
+
+The Economist Data Team represents the data in an elegant and simple style. The article's main graph is accompanied by an interactive table providing information about the cities, including their population and active mobility rate.
+
+I chose this graph because it seems an original way of presenting information on three variables. Besides, I am also really interested in issues related to urban mobility and possible differences between regions. The full article is available at this [link](https://www.economist.com/interactive/2024-walkable-cities).
+
+### Original chart
+
+The original graphic is an interactive ternary graphic in which a distinctive colour represents the three types of mobility mentioned above. In addition, when hovering over each point, the city's name associated with that point is indicated. The graph contains commentary, so as you scroll down, it is divided into different continents, and the article comments on the most salient results.
+
+![](images/graph.png){width="538"}
+
+## Replication
+
+### Libraries
+
+The first step is to load the libraries that will be used.
+
+```{r, warning = FALSE}
+
+library(tidyverse)
+library(ggthemes)
+library(scales)
+library(magick)
+library(ggtern)
+library(patchwork)
+library(plotly)
+library(lemon)
+library(shiny)
```

Comment:
Please remove shiny here, because you cannot use it in the post. In general, it is a good practice to remove any library that is not directly called in your code, so that another person trying to replicate this can install only the minimal set of packages required. E.g. are you using magick? (Didn't check).